web/src/components/OrganizationContentInfo.tsx

```diff
@@ -16,6 +16,7 @@ const StyledImage = styled.img<{ viewportSmall: boolean }>`
 
 const ThumbnailSizer = styled.div`
   width: 150px;
+  margin-right: 20px;
```

Comment:
We should avoid css set to left/right bc it breaks on RTL layouts. Can you figure out some other way?

Reply:
Arg, good point, thanks!